tools/payloads/configbytes/apk_configbytes_extract.rb

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env ruby
+# -*- coding: binary -*-
+
+class ExtractError < StandardError; end
+class HelpError < StandardError; end
+class UsageError < ExtractError; end
+
+require 'rex'
```

Comment:
I get an exception when I try to run this. It looks like it's missing the library path. Should be able to fix that by adding this:

```suggestion
msfbase = __FILE__
while File.symlink?(msfbase)
  msfbase = File.expand_path(File.readlink(msfbase), File.dirname(msfbase))
end

$:.unshift(File.expand_path(File.join(File.dirname(msfbase), '..', '..', '..', 'lib')))

require 'rex'
```